core-tests/jvm/src/test/scala/zio/IOSpec.scala

```diff
@@ -305,6 +307,21 @@ class IOSpec(implicit ee: org.specs2.concurrent.ExecutionEnv) extends TestRuntim
     )
   }
 
+  def testCached = {
+    def incrementAndGet(ref: Ref[Int]): UIO[Int] = ref.modify(n => (n + 1, n + 1))
```

Comment:
Using `update` instead of `modify`, you won't have to repeat `n + 1`

Reply:
Yes.